package.json

```diff
@@ -46,7 +46,7 @@
     "build:web": "GENERATE_SOURCEMAP=false react-scripts build",
     "build:api": "webpack --config webpack.api.config.js --mode production",
     "build:api:watch": "npm run build:api -- --watch --preserveWatchOutput",
-    "test": "react-scripts test",
+    "test": "NODE_OPTIONS=--max-old-space-size=2048 react-scripts test",
```

Comment:
uten `max-old-space-size=2048` så feiler noen av testene på jenkins med `ENOMEM: not enough memory, read`, og det funker heller ikke lokalt å kjøre testene.